code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -39,6 +39,17 @@
 	can_unsuppress = 0
 	can_suppress = 0
 
+/obj/item/gun/projectile/automatic/sniper_rifle/compact //holds very little ammo, lacks zooming, and bullets are primarily damage dealers, but the gun lacks the downsides of the full size rifle
+	name = "compact sniper rifle"
+	desc = "a compact, non-scopable version of the standard issue syndicate sniper rifle. Still capable of sending people crying."
```

Comment:
Capital A

Reply:
Also, 'scopable' isn't a word. Unscoped is. I'd personally change it to:

> A compact, unscoped version of the standard issue syndicate sniper rifle without . Still capable of sending people flying.

or

>A compact version of the standard issue syndicate sniper rifle which cannot use optics. Still capable of sending people flying.

Reply:
Unscoped would make more sense, yeah.